CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 ### Bug Fixes
 
 - Fixed PolicyGroup creation to include required entityType and mode fields [#563](https://github.com/pulumi/pulumi-pulumiservice/issues/563)
+- Removed references to deactivated user from approval rule examples
```

Comment:
We don't need this change in CHANGELOG.md, since its neither a bug fix nor an enhancement for users of this  provider.